### PR TITLE
Add print task definition resource to graph webhook docs

### PR DESCRIPTION
**Pull request by jahsu-MSFT**

### CODE REVIEW

File: api-reference/beta/api/subscription-delete.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ Depending on the resource and the permission type (delegated or application) req
 |[presence](../resources/presence.md) | Presence.Read.All | Not supported | Not supported |
 |[security alert](../resources/alert.md) | SecurityEvents.ReadWrite.All | Not supported | SecurityEvents.ReadWrite.All |
 |[user](../resources/user.md) | User.Read.All | User.Read.All | User.Read.All |
+|[printTaskDefinition](../resources/printtaskdefinition.md) | Not supported | Not supported | PrintTaskDefinition.ReadWrite.All |
```
Comment (baywet) [inline api-reference/beta/api/subscription-delete.md:40]: can you move this line so objects read in alphabetical order please?

**Review by baywet — APPROVED** [review]

**Comment by jahsu-MSFT** [comment]
Please don't merge this PR yet until the end of next week when all the updates are deployed.

**Comment by baywet** [comment]
I don't have merge permissions on this repo :)
I've added the "do not merge" tag. Once you're ready to merge, remove that tag and add the "ready to merge" one instead.

**Comment by jahsu-MSFT** [comment]
> I don't have merge permissions on this repo :)
> I've added the "do not merge" tag. Once you're ready to merge, remove that tag and add the "ready to merge" one instead.

Wasn't aware of this tag. That sounds great. Thanks!

**Comment by Lauragra** [comment]
@jahsu-MSFT , please set the ***ready to merge** label on the PR when you're ready.

**Comment by jahsu-MSFT** [comment]
> @jahsu-MSFT , please set the ***ready to merge** label on the PR when you're ready.

Will do! Thanks for the note.

**Review by baywet — CHANGES_REQUESTED** [review]
I just noticed you forgot to fill in the max expiration time https://docs.microsoft.com/en-us/graph/api/resources/subscription?view=graph-rest-beta#maximum-length-of-subscription-per-resource-type

**Comment by jahsu-MSFT** [comment]
> I just noticed you forgot to fill in the max expiration time https://docs.microsoft.com/en-us/graph/api/resources/subscription?view=graph-rest-beta#maximum-length-of-subscription-per-resource-type

@baywet Thought I did. It's in the subscription.md file right? | Print **printTaskDefinition** | 4230 minutes (under 3 days)    |

**Comment by baywet** [comment]
@jahsu-MSFT right, sorry I missed it cross checking with the checklist

**Review by baywet — APPROVED** [review]

**Review by andrewchoi5 — APPROVED** [review]

**Comment by jahsu-MSFT** [comment]
@baywet Thanks for fixing the conflicts! @Lauragra The latest changes are deployed to prod and this PR is now ready to be merged in. 
Thanks,

**Review by Lauragra — APPROVED** [review]

**Pull request merged**